components/brave_player/resources/player.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Brave Player</title>
+<link rel="stylesheet" href="chrome://resources/brave/fonts/poppins.css">
+<link rel="stylesheet" href="chrome://resources/brave/leo/css/variables.css">
+<script src="/player.bundle.js"></script>
+<style>
+  #root { 
+    position: fixed;
+    width: 100%;
```

Comment:
Any reason we don't use `100vw` and `100vh` here?

Reply:
nah, thanks!